examples/src/examples/user-interface/text-auto-font-size.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import * as pc from 'playcanvas/build/playcanvas.js';
+import Example from '../../app/example';
+import { AssetLoader } from '../../app/helpers/loader';
+
+class TextAutoFontSizeExample extends Example {
+    static CATEGORY = 'User Interface';
+    static NAME = 'Text Auto Font Size';
+
+    load() {
+        return <>
+            <AssetLoader name='font' type='font' url='static/assets/fonts/courier.json' />
+        </>;
+    }
+
+    example(canvas: HTMLCanvasElement, assets: { font: pc.Asset }): void {
+
+        // Create the application with input and start the update loop
+        const app = new pc.Application(canvas, {
+            mouse: new pc.Mouse(document.body),
+            touch: new pc.TouchDevice(document.body),
+            elementInput: new pc.ElementInput(canvas)
+        });
+        app.start();
+
+        // Set the canvas to fill the window and automatically change resolution to be the same as the canvas size
+        app.setCanvasFillMode(pc.FILLMODE_FILL_WINDOW);
+        app.setCanvasResolution(pc.RESOLUTION_AUTO);
+
+        window.addEventListener("resize", function () {
+            app.resizeCanvas(canvas.width, canvas.height);
+        });
+
+        // Create a camera
+        const camera = new pc.Entity();
+        camera.addComponent("camera", {
+            clearColor: new pc.Color(30 / 255, 30 / 255, 30 / 255)
+        });
+        app.root.addChild(camera);
+
+        // Create a 2D screen
+        const screen = new pc.Entity();
+        screen.addComponent("screen", {
+            referenceResolution: new pc.Vec2(1280, 720),
```

Comment:
just a note to check / make sure the example work correctly on mobile in portrait orientation as well.

Reply:
Yea good point! The examples themselves are working just fine. But the entire layout is a bit weird:

<img width="1689" alt="Screenshot 2021-12-01 at 01 42 25" src="https://user-images.githubusercontent.com/6392953/144156479-6a72add0-1364-4404-ab92-b3e25584d8bd.png">